eZ/Publish/Core/FieldType/Image/ImageStorage.php

```diff
@@ -73,6 +79,7 @@ public function storeFieldData(VersionInfo $versionInfo, Field $field, array $co
                 ),
                 $field->value->externalData['fileName']
             );
+//            $targetPath = $this->filePathNormalizer->normalizePath($targetPath);
```

Comment:
Why commented?

Reply:
> Why commented?

Good point. We don't keep dead code.

Reply:
I've commented this out for testing, forgot to uncomment. Good catch @glye